vue/src/store/generated/mande-chain/mandechain.voting/module/types/cosmos/base/query/v1beta1/pagination.ts

```diff
@@ -38,12 +38,6 @@ export interface PageRequest {
    * is set.
    */
   count_total: boolean;
-  /**
-   * reverse is set to true if results are to be returned in the descending order.
-   *
-   * Since: cosmos-sdk 0.43
-   */
-  reverse: boolean;
```

Comment:
I know these are generated changes but not sure if we should push these
We might need to test the UI properly so that none of the APIs break with these changes

Reply:
Ideally, the generated changes should not break the UI. 
Also, I've tested it to not break existing functionality.

Reply:
Will close this PR and create a new one without including UI changes.